_includes/gc-follow-us.html

```diff
@@ -1,9 +1,17 @@
 <section class="provisional gc-followus">
 	<h2>
-		{%- if i18nText-lang == "fr" -%}
-			Dans les médias sociaux
+		{%- if include.inlineLayout -%}
+			{%- if page.language == "fr" -%}
+				Suivez-nous
+			{%- else -%}
+				Follow us
+			{%- endif -%}
 		{%- else -%}
-			On social media
+			{%- if page.language == "fr" -%}
+				Dans les médias sociaux
+			{%- else -%}
+				On social media
+			{%- endif -%}
 		{%- endif -%}
 	</h2>
 	<ul{% if page.followUsLayout == "inline" %} class="list-inline"{% endif %}>
```

Comment:
Don't forget to adjust it here too:
```suggestion
	<ul{% if include.isInline %} class="list-inline"{% endif %}>
```